docs/en/operations/system-tables/settings_profiles.md

```diff
@@ -0,0 +1,20 @@
+# system.settings_profiles {#system_tables-settings_profiles}
+
+Collection of settings for authorized users. 
+
+Columns:
+-    `name` ([String](../sql-reference/data-types/string.md)) — Setting profile name.
+
+-    `id` ([UUID](../sql-reference/data-types/uuid.md)) — Setting profile ID.
+
+-    `storage` ([String](../sql-reference/data-types/string.md)) — Path to the storage of setting profiles. Configured in the `access_control_path` parameter. 
+
+-    `num_elements` ([UInt64](../sql-reference/data-types/int-uint.md)) — 
```

Comment:
Number of elements for this profile in the `system.settings_profile_elements` table.